src/psbt.h

```diff
@@ -479,6 +692,7 @@ struct PartiallySignedTransaction
                     // Make sure that all scriptSigs and scriptWitnesses are empty
                     for (unsigned int i = 0; i < tx->vin.size(); i++) {
                         const CTxIn& txin = tx->vin[i];
+                        tx->witness.vtxinwit.resize(tx->vin.size());
```

Comment:
resize it outside the loop?

Reply:
Heh, that would be clever. I think I did this in the rebase.

Reply:
Done.